docker-compose.staging.yaml

```diff
@@ -4,8 +4,13 @@ services:
 
   app:
     image: usedgoodreads/app:staging
-    ports:
-      - "8080:5000"
 
   worker:
     image: usedgoodreads/app:staging
+
+  nginx:
+    volumes:
+      - /etc/letsencrypt/live/usedgoodreads.com/:/certs
+    ports:
+      - 8080:80
+      - 4443:443
```

Comment:
Have you tried this actually? How does the browser know to request https from 4443?

Reply:
I requested `localhost:4443` directly